app/jobs/generate_payout_report_job.rb

```diff
@@ -19,6 +19,28 @@ def perform(should_send_notifications: true, final: true)
       publisher_has_unsettled_balance = false
       wallet = publisher.wallet
 
+      probi = wallet.channel_balances[publisher.owner_identifier].probi_before_fees # probi = balance
```

Comment:
We're calling this function down below and there seems to be some duplicate logic, is there a way to simplify this and combine these two branches of logic to make it more DRY?

Reply:
Ideally we would do as you describe and organize the loop such that we iterate through all the `wallet.channel_balances` (which is really account balances, i.e. channel balances + owner balance) instead of once for the owner, and then once for each channel.  However there are some other problems which make that difficult.

I think the logic here is different enough that it justifies the repetition.